src/services/signer/stellar/local_signer.rs

```diff
@@ -0,0 +1,155 @@
+use crate::{
+    domain::{
+        SignDataRequest, SignDataResponse, SignTransactionResponse, SignTransactionResponseStellar,
+        SignTypedDataRequest,
+    },
+    models::{Address, NetworkTransactionData, SignerError, SignerRepoModel},
+    services::Signer,
+};
+use alloy::primitives::FixedBytes;
+use async_trait::async_trait;
+use ed25519_dalek::Signer as Ed25519Signer;
+use ed25519_dalek::{ed25519::signature::SignerMut, SigningKey};
+use eyre::Result;
+use sha2::{Digest, Sha256};
+use soroban_rs::Signer as SorobanSigner;
+use std::convert::TryInto;
+use stellar_xdr::curr::{
+    DecoratedSignature, Hash, Limits, Signature, SignatureHint, Transaction, TransactionEnvelope,
+    TransactionSignaturePayload, TransactionSignaturePayloadTaggedTransaction,
+    TransactionV1Envelope, VecM, WriteXdr,
+};
+
+pub struct LocalSigner {
+    local_signer_client: SorobanSigner,
+}
+
+impl LocalSigner {
+    pub fn new(signer_model: &SignerRepoModel) -> Result<Self, SignerError> {
+        let config = signer_model
+            .config
+            .get_local()
+            .ok_or_else(|| SignerError::Configuration("Local config not found".into()))?;
+
+        let local_signer_client = {
+            let key_bytes = config.raw_key.borrow();
+            SorobanSigner::new(SigningKey::from_bytes(&FixedBytes::from_slice(&key_bytes)))
```

Comment:
Can we avoid using FixedBytes?

Here is Marcos example

https://github.com/MCarlomagno/keystore/blob/main/examples/local-keystore-to-stellar-wallet/src/main.rs